torch_xla/csrc/elementwise.cpp

```diff
@@ -76,6 +76,42 @@ xla::XlaOp BuildRelu(const xla::XlaOp& input) {
                              0, input_shape.element_type(), input.builder()));
 }
 
+xla::XlaOp BuildHardtanh(const xla::XlaOp& input, at::Scalar min_val,
+                         at::Scalar max_val) {
+  xla::Shape shape = XlaHelpers::ShapeOfXlaOp(input);
+  xla::PrimitiveType element_type = shape.element_type();
+  xla::XlaBuilder* builder = input.builder();
+  xla::XlaOp min_val_branch =
```

Comment:
We do have Clamp() in ops.*, which maps directly to xla::Clamp().

Reply:
Done, but note there's a behavior difference between PyTorch `clamp` / `hardtanh` (they are synonymous AFAICT) and our `clamp` when `min_val` is greater than `max_val`. Ours returns `max_val` everywhere, PyTorch returns `min_val` for values which are smaller than `min_val`. Arguably this is incorrect input anyway and we can pick a different behavior, but I'm pointing out there *is* a difference if we use `xla::Clamp`.